Sources/main.swift

```diff
@@ -12,7 +12,7 @@ import SwiftGenKit
 
 // MARK: Common
 
-let templatesRelativePath = (Bundle.main.resourcePath ?? ".") + "/templates"
+let templatesRelativePath = "../templates"
```

Comment:
Why this change? Doesn't this not have to work both in Xcode (who building a .app bundle) and in CLI (once split and the templates/resources are next to the binary outside an app bundle)?